src/App.jsx

```diff
@@ -1,28 +1,141 @@
+/* eslint-disable react/no-access-state-in-setstate */
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import classNames from 'classnames';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
+    copiedTodos: [],
     selectedUserId: 0,
+    title: '',
+    selectOption: '',
```

Comment:
```suggestion
    selectedOption: '',
```
selectOption - звучит как название функции